html-forms_personal-profile-form/index.html

```diff
@@ -10,6 +10,35 @@
     <link rel="stylesheet" href="./css/styles.css" />
   </head>
   <body>
-    <section></section>
+    <section>
+      <form action="submit">
+      <p>We want to get to know you! Please enter your personal information</p>
+      <fieldset name="Peronal Information">
```

Comment:
Just a nit: the only confusing thing for me is that this fieldset has a name but the next one doesn't. 

another nit is that [we can tell the screen reader these two fieldsets belong to the same form](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset)